package.json

```diff
@@ -10,7 +10,13 @@
   "scripts": {
     "server-dev": "nodemon server",
     "react-dev": "webpack -d --watch",
-    "seed": "node db/dbSeed.js"
+    "seed": "node db/dbSeed.js",
+    "test": "jest"
+  },
+  "jest": {
+    "transform": {
+      "^.+\\.jsx?$": "babel-jest"
+    }
```

Comment:
What is this for?

Reply:
I'm curious where you found that also. Looks like it helps tell jest to send javascript and JSX files through the babel-jest plugin first, but I didn't come across this on my install.

Reply:
I found that in the babel docs if you click on 'jest' under the test frameworks
https://babeljs.io/setup#installation